Applications/CommandLineInterface/src/selxSuperElastix.cxx

```diff
@@ -72,6 +72,8 @@ main( int ac, char * av[] )
 {
 
   selx::Logger::Pointer logger = selx::Logger::New();
+  logger->AddStream( "cout", std::cout );
```

Comment:
duplicate functionality due to merge: https://github.com/SuperElastix/SuperElastix/blob/develop/Applications/CommandLineInterface/src/selxSuperElastix.cxx#L120-L121

Reply:
With respect the name, you put `SELXcout`. But the `cout` _is_ special, there is only one, so perhaps it is misleading to prepend the name with `SELX`, since, strictly speaking, it is not _our_ cout?

Reply:
Yes you are right. Make it cout.